### ISSUE TITLE
Doubt about the update (0.113.0)

### ISSUE THREAD

**Issue by jvirriel-gekocloud** [open-telemetry/opentelemetry-collector#11690]
Hi! 
We do a update from 0.104.0 to 0.113.0 the last friday and I have a problem with a certificate, in the operator manifest file, specifically in the deployment "opentelemetry-operator-controller-manager" that we downloaded there is the declaration of a volume for a certificate:
volumeMounts:
- mountPath: /var/run/tls/server
name: opentelemetry-operator-metrics-cert
serviceAccountName: opentelemetry-operator-controller-manager
terminationGracePeriodSeconds: 10
volumes:
- name: opentelemetry-operator-metrics-cert
secret:
defaultMode: 420
secretName: opentelemetry-operator-metrics

But the creation of this certificate is not present in the manifest, as if it is:

- name: cert
secret:
defaultMode: 420
secretName: opentelemetry-operator-controller-manager-service-cert

However, I have been reviewing the github manifest again and now it does not appear the volume for the metrics certificate, is this a bug? Has a fix been uploaded? We manually created the certificate manifest file in order to fix the problem, but I'm not sure if it was correct, or there is a bug in the manifest of the current version (0.113.0)

**Comment by codeboten** [comment]
I think this issue should be transferred to the operator. maybe @jaronoff97 or @dmitryax can help

**Comment by jaronoff97** [comment]
The certificate should be created with that name for the main cert [here](https://github.com/open-telemetry/opentelemetry-operator/blob/3d254a47121385bc9de832f37e9767956d2145c3/config/certmanager/certificate.yaml#L13-L29). The metrics-cert is only installed for the [openshift bundle](https://github.com/open-telemetry/opentelemetry-operator/blob/3d254a47121385bc9de832f37e9767956d2145c3/bundle/openshift/manifests/opentelemetry-operator.clusterserviceversion.yaml#L540) and is handled by the OLM I believe. If you are installing this via openshift I would bubble up the issue to them, if you are not openshift you should be installing via the community bundle or via our helm-chart. Either way, this does make sense to move to the operator, though I do not have permissions to do so 😅

**Comment by codeboten** [comment]
@bogdandrutu can you transfer the issue

**Comment by codeboten** [comment]
@jaronoff97 is this issue still relevant? If so it might be worth opening a separate issue for it

**Comment by jaronoff97** [comment]
Nope, not relevant anymore. I think we delete this. The problem was related to openshift, the stuff we ship for community manifests does not have this.
